database/04-seeds-and-factories.adoc

```diff
@@ -58,12 +58,18 @@ class DatabaseSeeder {
 module.exports = DatabaseSeeder
 ----
 
-.Running The Seed Command
+.Running The Seed Command - _for all seeds_
```

Comment:
The formatting is so different from what we have in the docs till now. Would you mind updating it as defined below

1. Let the heading **Running the Seed Command** as it is
2. After the 1st code example, create a simple paragraph saying. `Run the following command to execute a single seed file`